kedro/config/omegaconf_config.py

```diff
@@ -357,6 +358,18 @@ def load_and_merge_dir_config(  # noqa: PLR0913
             if not k.startswith("_")
         }
 
+    def _get_all_keys(self, cfg: dict, parent_key: str = "") -> set[str]:
+        keys: set[str] = set()
+
+        for k, v in cfg.items():
```

Comment:
Can you use more descriptive variable names? In the Kedro codebase we aim to keep code as readable and understandable as possible, so instead of generic names like `k` and `v` and abbreviations it's preferred to name things explicitly.